docs/immortal-empires/kislev.md

```diff
@@ -0,0 +1,32 @@
+# Kislev
+
+## Kostaltyn: The Great Orthodoxy
+
+* **Route I - There Can Only Be One**
+  * PLACEHOLDER:
```

Comment:
Placeholders have to be completed.

Reply:
Titles and flavour text created, added to docs and tables.